update-scripts/update.nix

```diff
@@ -43,7 +43,7 @@ writeShellApplication {
 
     versionInfo() {
       nix-build ./update-scripts -A version-info
-      install -m 644 -T "$(realpath result)" ./version-info.toml
+      ./result
```

Comment:
Oops, this is a directory. The script is actually at `./result/bin/version-info`